### PR TITLE
Changelog and version for Composer / Packagist

### PR DESCRIPTION
**Pull request by Steve-Fenton**
Preparation to publish a 1.x version prior to removing support for old PHP verisons.

See #41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,13 @@ A unit testing framework with mocks and stubs. Built for PHP, in PHP!
 
 Quick Start: Just add `EnhanceTestFramework.php` and you are ready to go. 
 
-[Check out the documentation](https://github.com/Enhance-PHP/Enhance-PHP/wiki) for more: 
+- [Documentation](https://github.com/Enhance-PHP/Enhance-PHP/wiki) for more: 
+- [View the change log](https://github.com/Enhance-PHP/Enhance-PHP/blob/main/CHANGELOG.md)
```
Comment (tbreuss) [inline README.md:8]: I would write "changelog" in one word like in CHANGELOG.md

File: README.md [hunk 2]
```diff
@@ -4,10 +4,13 @@ A unit testing framework with mocks and stubs. Built for PHP, in PHP!
...
 
 ## Supported PHP Versions
 
+We track [officially supported PHP versions]([officially supported PHP versions](https://www.php.net/supported-versions.php).
```
Comment (tbreuss) [inline README.md:12]: The link has an incorrect markdown syntax.

**Review by tbreuss — COMMENTED** [review]
I found just 2 typos.

I'm looking forward to the official release :)

**Comment by Steve-Fenton** [comment]
Good spot! I think I've fixed it all up.

**Review by tbreuss — APPROVED** [review]

**Pull request merged**